crates/goose/src/config/signup_tetrate/mod.rs

```diff
@@ -14,7 +14,7 @@ use tokio::sync::oneshot;
 use tokio::time::timeout;
 
 /// Default models for Tetrate Agent Router Service configuration
-pub const TETRATE_DEFAULT_MODEL: &str = "claude-4-sonnet-20250514";
+pub const TETRATE_DEFAULT_MODEL: &str = "claude-haiku-4-5";
```

Comment:
Did you mean to add this change @Rodriguespn ? Perhaps you pulled this in by accident. After that, i can approve and merge